src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +50,22 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'The name of variable to filter for debug.'),
```

Comment:
[Since 6.1, the suggested values can be defined in 5th argument.](https://symfony.com/blog/new-in-symfony-6-1-improved-console-autocompletion#completion-values-in-input-definitions)
```suggestion
                new InputArgument('name', InputArgument::OPTIONAL, 'The name of variable to filter for debug.', null, $this->getAvailableVars(...)),
```

Reply:
changed